code/datums/dna.dm

```diff
@@ -613,7 +613,7 @@ GLOBAL_LIST_INIT(total_uf_len_by_block, populate_total_uf_len_by_block())
 	if(dna.features["pod_hair"])
 		dna.features["pod_hair"] = GLOB.pod_hair_list[deconstruct_block(get_uni_feature_block(features, DNA_POD_HAIR_BLOCK), GLOB.pod_hair_list.len)]
 
-	for(var/obj/item/organ/external/external_organ as anything in external_organs)
+	for(var/obj/item/organ/external/external_organ in internal_organs)
```

Comment:
100% in the future `internal organs` has to be renamed but for the sake of keeping prs reviewable and moving i'll let it slide

Reply:
I'll do it tomorrow

Reply:
no wait Ill do it after fikou finishes his organ species rework